lib/roo_on_rails/railties/database.rb

```diff
@@ -1,7 +1,7 @@
 module RooOnRails
   module Railties
     class Database < Rails::Railtie
-      if ActiveRecord::VERSION::MAJOR >= 4
+      if defined?(ActiveRecord) && ActiveRecord::VERSION::MAJOR >= 4
```

Comment:
Something like that I guess... https://github.com/sideshowcoder/activerecord_translatable/blob/master/lib/activerecord_translatable.rb#L10-L18 (oh boy that was a long time ago please don't look at the other code)